_migrate-or-upgrade/rolling-upgrade/index.md

```diff
@@ -1,19 +1,23 @@
 ---
 layout: default
 title: Rolling upgrade
-parent: Upgrading OpenSearch
-nav_order: 10
+nav_order: 20
+has_toc: true
+permalink: /migrate-or-upgrade/rolling-upgrade/
+redirect_from: 
+ - /upgrade-opensearch/appendix
+ - /rolling-upgrade/index
 ---
 
 # Rolling upgrade
```

Comment:
This page has 2 headings level 1: Rolling upgrade and Rolling restart. Please make Rolling restart and Related articles headings level 2.

Reply:
For rolling restart, add links to the mentioned rolling upgrade steps.